site/tailwind.config.js

```diff
@@ -8,7 +8,8 @@ module.exports = {
 	important: ["#root", "#storybook-root"],
 	theme: {
 		fontSize: {
-			"2xs": ["0.626rem","0.875rem"],
+			"2xs": ["0.626rem", "0.875rem"],
+			xs: ["0.75rem", "1rem"],
```

Comment:
"0.75rem", "1rem" is already the Tailwind default for xs so this should not be necessary

Reply:
It wasn't working until I added it. Can you test it on your end to double-check?